models/public/efficientnet-b0/efficientnet-b0.md

```diff
@@ -2,11 +2,11 @@
 
 ## Use Case and High-Level Description
 
-The `efficientnet-b0` model is one of the [EfficientNet](https://arxiv.org/abs/1905.11946)
-group of models designed to perform image classification.
+The `efficientnet-b0` model is one of the [EfficientNet](https://arxiv.org/abs/1905.11946) models 
+designed to perform image classification.
 This model was pretrained in TensorFlow\*.
-All the EfficientNet models have been pretrained on the ImageNet image database.
-For details about this family of models, check out the [repository](https://github.com/tensorflow/tpu/tree/master/models/official/efficientnet).
+All the EfficientNet models have been pretrained on the ImageNet\* image database.
+For details about this family of models, check out the [TensorFlow EfficientNet repository](https://github.com/tensorflow/tpu/tree/master/models/official/efficientnet).
```

Comment:
Strictly speaking, this isn't the _EfficientNet_ repository. It is the TensorFlow Cloud TPU repository (which is where the EfficientNet models happen to be).

Reply:
Please update the other links to this URL as well.